frontend/src/components/Modal/RiderModalInfo.tsx

```diff
@@ -165,7 +165,7 @@ const RiderModalInfo = ({
             aria-required="true"
             ref={register({
               required: true,
-              pattern: /^[a-zA-Z0-9\s,.'-]{3,}$/,
+              validate: (value) => value.trim() !== '',
```

Comment:
Hi Raissa, you did a great job on this PR as your code was both simple and correct. Just a sidenote, but if you wanted to do this with regex, you could use: `pattern : /^(?!\s*$).+/`, which represents strings with at least one non empty character.